### ISSUE TITLE
mobile viewer

### ISSUE THREAD

**Issue by thesamovar** [thesamovar/notpaper#11]
How easy is it to make an html/js/css app into an Android/iOS app? I'm thinking a pdf viewer built from pdf.js that has some of the capabilities of this built into it.

**Comment by rorybyrne** [comment]
React has a sister-library [React Native](https://reactnative.dev/) for the creation of Android/iOS apps. I haven't used it though (or done any mobile app development really).

I think it sounds reasonable to isolate some of this project's codebase so that it can be re-used in a mobile app project.

**Comment by thesamovar** [comment]
The way I see it working on mobile is that each time you click on a link, it creates a new view stacked to the right and moves to it. You can swipe left to go back to your previous view, or right to go forward to the last clicked point in the stack. Clicking a new item removes everything to the right in the stack.

This might also work for desktop actually as an alternative approach.

**Comment by rorybyrne** [comment]
The idea of a carousel that slides left-and-right is common, even in the regular HTML/JS/CSS world. 

It might make more sense to build this as a mobile-designed ("responsive") website, rather than a native mobile app. It would be faster to build and easier to distribute as well ("go to this link", rather than "install this app"). It's surprisingly difficult to convince people to install a new app, esp. because many people have literally no disk space left on their phone to install anything.

In-fact, this could just be the mobile version of the current 3-panel desktop website right?

**Comment by thesamovar** [comment]
Yes, that makes sense. I think the interaction should be slightly different on mobile compared to desktop because on mobile you'd want to only see one bit of information at one time, but make it very easy (swipe) to flip back and forth. On desktop you'd want to show both at the same time.
